test/core/pipeline/test_breakpoint.py

```diff
@@ -9,7 +9,7 @@
 from haystack.core.pipeline.breakpoint import (
     _transform_json_structure,
     _validate_break_point,
-    _validate_resume_state,
+    _validate_pipeline_snapshot,
     load_state,
```

Comment:
`load_snapshot` - already mentioned it above - just a reminder that  then tests also need to be updated